libcontainer/cgroups/fs/cpuset.go

```diff
@@ -5,6 +5,7 @@ package fs
 import (
 	"bytes"
 	"errors"
+	"fmt"
```

Comment:
I think you should remove "fmt" here.
And change `fmt.Errorf` to `errors.Errorf`.